config/ie/options

```diff
@@ -350,6 +350,11 @@ if targetApp=="houdini" :
 	
 	HOUDINI_ROOT = houdiniReg['location']
 	HOUDINI_CXX_FLAGS = " ".join( CXXFLAGS ) + ' -DVERSION=\"'+houdiniVersion+'\" -DDLLEXPORT=  -D_GNU_SOURCE -DLINUX -DAMD64 -m64 -fPIC -DSIZEOF_VOID_P=8 -DSESI_LITTLE_ENDIAN -DENABLE_THREADS -DUSE_PTHREADS -D_REENTRANT -D_FILE_OFFSET_BITS=64 -DGCC4 -DGCC3 -Wno-deprecated -Wno-reorder -std=c++11'
+	# Houdini 16.0 and beyond can optionally ship using Qt5.
+	# Since IECoreHoudini makes some UI related calls, we add
+	# a custom define so we can change the logic as needed.
+	if int(houdiniReg.get( "qtVersion", "4" ).split( "." )[0]) >= 5 :
+		HOUDINI_CXX_FLAGS += " -DIECOREHOUDINI_WITH_QT5"
```

Comment:
It seems unfortunate that we're not providing a way of deriving this information that works outside of Image Engine.

Reply:
They don't ship the Qt5 headers unfortunately, so I can't use `CheckLibWithHeader` in the SConstruct. Maybe there's a `CheckLib` I could use instead though... I'll have a look.

Reply:
Addressed in 4c76829c5a4ee58df10e1d1473f68258656dc75d